### DISCUSSION TITLE
Documentation about lisp is obsolete

### DISCUSSION THREAD

**Discussion by gerchicov-bp** [Chia-Network/chia-blockchain#5508] [Support]
from your readme:
>Lisp-style turing complete functional programming language for money related use cases
https://github.com/Chia-Network/clvm/issues/87

If your lisp has issues then ok, those issues belong to its own repo. BUT why do you post links to the resources (including that repo) which became obsolete?

**Comment by BrandtH22** [comment]
Hey @gerchicov-bp , since this ticket was opened a number of years ago without further comments we will be closing it out. 

If you are still looking for the latest information on chialisp I recommend looking at the https://chialisp.com/ site. 

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
